lib/client/node.js

```diff
@@ -311,6 +311,15 @@ class NodeClient extends Client {
 
     return this.call('rescan', start);
   }
+
+  /**
+   * Abort scanning blockchain
+   * @returns {Promise}
+  */
```

Comment:
nit: ah so close, sorry :-P

```suggestion
   */
```